src/main/java/net/glowstone/inventory/GlowInventory.java

```diff
@@ -200,6 +200,7 @@ public ItemStack tryToFillSlots(ItemStack stack, int... slots) {
         }
         ItemStack maxStack = stack.clone();
         maxStack.setAmount(stack.getMaxStackSize());
+        Integer firstEmpty = null;
```

Comment:
Could this be an `int` instead? Just use -1 for an uninitialized value.

Reply:
Are there any instances where the slot number could be -1?